src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -202,7 +203,7 @@ HttpStatusCode mongoSubscribeContext
     }
 
     /* Adding format to use in notifications */
-    sub.append(CSUB_FORMAT, "JSON");
+    sub.append(CSUB_FORMAT, notificationFormatToString(requestP->attrsFormat));
```

Comment:
I looked them up:

1. serviceRoutines/postSubscribeContext.cpp:
```
    if (ciP->version != "v2")
    {
      parseDataP->scr.res.attrsFormat = NGSI_V1_JSON;
    }
```
2. serviceRoutines/postUpdateContextSubscription.cpp:
```
    if (ciP->version != "v2")
    {
      parseDataP->ucsr.res.attrsFormat = NGSI_V1_JSON;
    }
```
3. serviceRoutinesV2/postSubscriptions.cpp:
```
    if (parseDataP->scr.res.attrsFormat == NGSI_NO_NOTIFICATION_FORMAT)
    {
      parseDataP->scr.res.attrsFormat = NGSI_V2_NORMALIZED;
    }
```
1: Only set if v1.
2: Only set if v1.
3: Only set if previously unset (and if v2)

Nothing to worry about